### PR TITLE
Stories Feature

### PR DESCRIPTION
**Pull request by ykl1**

### CODE REVIEW

File: src/static/styles/App.scss [hunk 1]
```diff
@@ -660,3 +674,99 @@ hr {
 	background-color: #6129a8;
 }
 
+.stories-sizing {
+	box-sizing: border-box;
+	font-size: 62.5%;
+}
+
+.stories-body {
+	font-family: "Poppins", sans-serif;
```
Comment (crchong1) [inline src/static/styles/App.scss]: can we stick to Inter?

File: src/static/styles/App.scss [hunk 2]
```diff
@@ -660,3 +674,99 @@ hr {
...
+}
+
+.card-content {
+	padding: 1rem;
+}
+
+.card-header {
+	font-size: 1.75rem;
+	font-weight: 520;
+	color: #0d0d0d;
```
Comment (crchong1) [inline src/static/styles/App.scss]: I would move all colors to a variable, and declare them at the top

File: src/static/styles/App.scss [hunk 3]
```diff
@@ -660,3 +674,99 @@ hr {
...
+	color: #6d6d6d;
+}
+
+.card-btn {
+	display: block;
+	width: 32.5%;
+	padding: 0.5rem;
+	font-size: 0.75rem;
+	text-align: center;
+	color: #fafafa;
```
Comment (crchong1) [inline src/static/styles/App.scss]: move color to variable

File: src/components/Stories.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import { Helmet } from 'react-helmet';
+import Card from './Card';
+import RectangleSVG from '../static/images/rectangle.svg';
+
+const url = 'https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/keep-informed';
+
+const Stories = () => {
+  const [mediumData, setMediumData] = useState([]);
+  useEffect(() => {
+    axios.get(url).then((response) => {
```
Comment (crchong1) [inline src/components/Stories.tsx]: do you have any additional thoughts on using axios? I know right now we usually use fetch, but is there a specific reason why you chose axios?

**Review by crchong1 — CHANGES_REQUESTED** [review]
see comments

File: src/components/Stories.tsx (hunk 4 above)
Comment (ykl1) [reply src/components/Stories.tsx]: I could use fetch! Axios has wider browser compatibility than fetch, and it automatically transforms responses into json data, which makes the code a bit cleaner than fetch.

**Review by crchong1 — APPROVED** [review]

**Pull request merged**